src/ui/skills.js

```diff
@@ -0,0 +1,282 @@
+import { CONSTANTS, FONTS } from "../constants/constants.js";
+import { calcLevel, calcRemainingXp } from "../utilities.js";
+import { getSkillDescription } from "./skill-descriptions.js";
+
+export class Skills {
+    scene;
+    statsScene;
+    saveData = {};
+    skillText = {};
+
+    hoverGraphics;
+    hoverWindow;
+    hoverNameText;
+    hoverLevelText;
+    hoverXpText;
+    hoverRemainingXpText;
+
+    skillInfo = {
+        bg: {},
+        header: {},
+        closeButton: {},
+        description: {},
+        body: {},
+    };
+
+    skillWidth = 64;
+    skillHeight = 31;
+
+    constructor(scene, skillData) {
+        this.scene = scene;
+        this.saveData = skillData;
+
+        let startX = 585,
+            startY = 220,
+            index = 0;
+
+        // Add level text for skills
+        for (let skill in this.saveData) {
+            let row = index % 3;
+            let column = Math.floor(index / 3);
+            let x = startX + row * this.skillWidth;
+            let y = startY + column * this.skillHeight;
+
+            this.skillText[skill] = scene.add
```

Comment:
I think these are slightly to the right of the actual skill boxes - or at least it seems so